packages/iTwinUI-react/src/core/Table/TablePaginator.tsx

```diff
@@ -282,7 +283,15 @@ export const TablePaginator = (props: TablePaginatorProps) => {
       ref={paginatorResizeRef}
       {...rest}
     >
-      <div className='iui-left' />
+      <div className='iui-left'>
+        {totalSelectedRowsCount > 0 && (
+          <span>
+            {`${totalSelectedRowsCount} ${
+              totalSelectedRowsCount === 1 ? 'row' : 'rows'
+            } selected`}
```

Comment:
this needs to be localized

Reply:
This is going to be fixed in an upcoming PR